.travis.yml

```diff
@@ -28,12 +28,14 @@ language: python
 sudo: required
 dist: bionic
 
+env:
+  global:
+    - MAKEFLAGS="-j 2"
 stages:
    - Staticcheck
-   - MPI-Threading-Python
-   - Python-Full-build
-   - Clang7
-   - OsX
+   - MPI-Threading-GSL-Python
+   - Python-Full-build-Clang7-OsX
```

Comment:
I don't quite like that we don't have a full build on Linux with GCC anymore. Is there a pressing reason for removing it?

Reply:
@jougs I think this is a misunderstanding. If you look at the [Travis builds for this PR](https://travis-ci.org/github/nest/nest-simulator/builds/682420651), the first build under this heading (#4501.6) is actually the full build with GCC. The label should be read as "Python full build + Clang build + OSX build, all running in parallel."

Reply:
@jougs Thanks for having a look. Sorry for the ambiguity in the naming of the build stage. I meant what @heplesser already mentioned. Thanks @heplesser

Reply:
@heplesser, @lekshmideepu: thanks for clarifying. In this case, I'm fine.